### PR TITLE
Updated MSAL

### PR DESCRIPTION
**Pull request by darrenjrobinson**
Updated MSAL to 2.26.0 to enable new MSAL features.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
-  <script src="https://alcdn.msauth.net/browser/2.15.0/js/msal-browser.js"
-    integrity="sha384-dFzMiVGB5HpWZ+5w5VSif6jhWfNeplSw9ACYmQKZcY2azuT9kCxVWVI9HyfGdkHV"
+  <script src="https://alcdn.msauth.net/browser/2.26.0/js/msal-browser.js"
+    integrity="sha384-fitpJWrpyl840mvd9nBFLGulqR4BJzvim0fzrXQKdsVh2AQzE4rTTJ0o5o+x+dRK"
     crossorigin="anonymous"></script>
 
   <!-- To help ensure reliability, Microsoft provides a second CDN -->
   <script type="text/javascript">
-    if (typeof Msal === 'undefined') document.write(unescape("%3Cscript src='https://alcdn.msftauth.net/browser/2.15.0/js/msal-browser.js' type='text/javascript' crossorigin='anonymous' %3E%3C/script%3E"));
+    if (typeof Msal === 'undefined') document.write(unescape("%3Cscript src='https://alcdn.msauth.net/browser/2.26.0/js/msal-browser.js' type='text/javascript' crossorigin='anonymous' %3E%3C/script%3E"));
```
Comment (derisen) [inline app/index.html]: `https://alcdn.msauth.net/` is the same CDN link as above, not a backup. Can we please keep the `alcdn.msftauth.net`?

**Review by derisen — CHANGES_REQUESTED** [review]
Much appreciated @darrenjrobinson , one comment to address

**Comment by darrenjrobinson** [comment]
Thanks @derisen 
I've updated the PR with an alternate CDN for updated MSAL version.

**Review by derisen — APPROVED** [review]
LGTM, thanks!

**Pull request merged**